<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> Project | Options </string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="project">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Definition</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QFormLayout" name="formLayout_7">
              <item row="0" column="1">
               <widget class="QLineEdit" name="name"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Name:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Description:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Author:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="description"/>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="author"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTabWidget" name="tabWidget">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>Transport</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QGroupBox" name="groupBox">
                  <property name="title">
                   <string>Convergence Criteria</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <layout class="QFormLayout" name="formLayout_3">
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="trans_iterations"/>
                     </item>
                     <item row="1" column="1">
                      <widget class="QLineEdit" name="trans_convergence"/>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>Convergence:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>Iterations:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="1">
                      <widget class="QLineEdit" name="trans_smoothing_factor"/>
                     </item>
                     <item row="2" column="0">
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>Smoothing Factor:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="1">
                      <widget class="QLineEdit" name="trans_route_similarity"/>
                     </item>
                     <item row="3" column="0">
                      <widget class="QLabel" name="label_7">
                       <property name="text">
                        <string>Route Similarity Factor %:</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>Land use</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QGroupBox" name="groupBox_2">
                  <property name="title">
                   <string>Convergence Criteria</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <item>
                    <layout class="QFormLayout" name="formLayout_5">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_13">
                       <property name="text">
                        <string>Iterations:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="land_iterations"/>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="label_12">
                       <property name="text">
                        <string>Convergence:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QLineEdit" name="land_convergence"/>
                     </item>
                     <item row="2" column="0">
                      <widget class="QLabel" name="label_14">
                       <property name="text">
                        <string>Smoothing Factor:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="1">
                      <widget class="QLineEdit" name="land_smoothing_factor"/>
                     </item>
                     <item row="3" column="1">
                      <widget class="QLineEdit" name="land_def_internal_cost_factor"/>
                     </item>
                     <item row="3" column="0">
                      <widget class="QLabel" name="label_8">
                       <property name="text">
                        <string>Default Internal Cost Factor %:</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
